specification/elasticsan/resource-manager/Microsoft.ElasticSan/stable/2023-01-01/elasticsan.json

```diff
@@ -0,0 +1,2391 @@
+      "get": {
+        "tags": [
+          "ElasticSan"
+        ],
+        "operationId": "ElasticSans_ListByResourceGroup",
+        "description": "Gets a list of ElasticSan in a resource group.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Operation completed successfully; retrieved elasticSans by resource group",
+            "schema": {
+              "$ref": "#/definitions/ElasticSanList"
+            }
+          },
+          "default": {
+            "description": "Error from the ElasticSan Resource Provider",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "ElasticSans_ListByResourceGroup_MaximumSet_Gen": {
+            "$ref": "./examples/ElasticSans_ListByResourceGroup_MaximumSet_Gen.json"
+          },
+          "ElasticSans_ListByResourceGroup_MinimumSet_Gen": {
+            "$ref": "./examples/ElasticSans_ListByResourceGroup_MinimumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ElasticSan/elasticSans/{elasticSanName}": {
+      "put": {
+        "tags": [
+          "ElasticSans"
+        ],
+        "operationId": "ElasticSans_Create",
+        "description": "Create ElasticSan.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ElasticSanNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ElasticSan"
+            },
+            "description": "Elastic San object."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Returned when the create operation is successfully completed.",
+            "schema": {
+              "$ref": "#/definitions/ElasticSan"
+            }
+          },
+          "201": {
```

Comment:
Is there a reason for the initial return code to be changed to 201 for create APIs? 
Is it the reason that ElasticSan create operations fall under the special case listed here? https://github.com/microsoft/api-guidelines/blob/vNext/azure/ConsiderationsForServiceDesign.md#put-with-additional-long-running-processing

Also from the doc https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#long-running-operations--jobs 
✅ DO return a 201-Created status code for create or 200-OK for replace from the initial request with a representation of the resource if the resource was created successfully.

Reply:
Hi @yifanz0, we changed the response to 201 as it was marked by LintDiff error, not sure if it is mandatory one. Can you please confirm if we can get exemption from it?

Reply:
Thanks for the explanation. No i was just wondering if there's a specific reason for the change, but it's up to the arm review team for the decision